ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopup.qml

```diff
@@ -78,14 +82,147 @@ ModalPopup {
         anchors.rightMargin: -Style.current.padding
     }
 
-    StyledText {
-        id: chatsTitle
-        text: qsTr("TODO:")
+    TextWithLabel {
+        id: shareCommunity
         anchors.top: sep1.bottom
         anchors.topMargin: Style.current.bigPadding
-        font.pixelSize: 15
-        font.weight: Font.Thin
+        label: qsTr("Share community")
+        text: "https://join.status.im/u/TODO"
+        textToCopy: text
     }
 
+    Separator {
+        id: sep2
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.top: shareCommunity.bottom
+        anchors.topMargin: Style.current.smallPadding
+        anchors.leftMargin: -Style.current.padding
+        anchors.rightMargin: -Style.current.padding
+    }
+
+    Column {
+        anchors.top: sep2.bottom
+        anchors.topMargin: Style.current.padding
+        width: parent.width
+        spacing: Style.current.padding
+        
+        
+        Loader {
+            active: isAdmin
+            width: parent.width
+            sourceComponent: CommunityPopupButton {
+                label: qsTr("Members")
+                iconName: "members"
+                txtColor: Style.current.textColor
+                Item {
+                    anchors.top: parent.top
+                    anchors.right: parent.right
+                    anchors.rightMargin: 0
+                    width: 100
+                    StyledText {
+                        text: nbMembers.toString()
+                        anchors.right: caret.left
+                        anchors.rightMargin: Style.current.smallPadding
+                        anchors.top: parent.top
+                        anchors.topMargin: 10
+                        padding: 0
+                        font.pixelSize: 15
+                        color: Style.current.secondaryText
+                    }
+
+                    SVGImage {
+                        id: caret
+                        anchors.right: parent.right
+                        anchors.topMargin: Style.current.padding
+                        anchors.top: parent.top
+                        source: "../../../img/caret.svg"
+                        width: 13
+                        height: 7
+                        rotation: -90
+                        ColorOverlay {
+                            anchors.fill: parent
+                            source: parent
+                            color: Style.current.secondaryText
+                        }
+                    }
+                }
+            }
+        }
+
+        Loader {
+            active: isAdmin
+            width: parent.width
+            sourceComponent: CommunityPopupButton {
+                label: qsTr("Roles")
```

Comment:
FYI, roles will not be in the first version most likely, but worst case, we can just put the `Loader` as `active: false`